bootstrap/ansible/main.copyme

```diff
@@ -159,6 +159,14 @@ sentry_dsn: ""
 # app_port: 8000
 # full_host_path: http://scgup-dev.lbl.gov:8000
 
+# # Storage settings.
+# # TODO: fill these out
+# file_storage_backend: 'google_drive'
```

Comment:
Are the following documented anywhere:
- What backends are available?
- How are the paths for each type of MOU constructed?
- `google_drive_private_key_file_path` is only needed if the Google Drive backend is being used (presumably).

Reply:
Should the default backend for development be local disk?

Reply:
![image](https://github.com/ucb-rit/coldfront/assets/43729496/58b8d227-4a05-45df-b14c-9ed0dd8bb056)
changed default to local disk and added these comments to `main.copyme`